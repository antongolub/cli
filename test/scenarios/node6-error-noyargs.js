'use strict'
global['NO_YARGS'] = true
require('../../')(main)
  .boolean('test')

function main () {
  console.log('started')
  return new Promise((resolve, reject) => {
    console.log('finished')
    resolve()
  })
}
