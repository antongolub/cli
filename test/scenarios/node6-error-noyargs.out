Error: Argument parsing is not available (could not find yargs), to install run: npm i yargs
    at Object.noYargs (app.js:)
1
